lib/api/routes/iterations.rb

```diff
@@ -11,17 +11,29 @@ class Iterations < Core
         require_key
 
         if current_user.guest?
```

Comment:
It's been a while since I looked at the related code, but I wonder if `require_key` means that you can't be a guest beyond that point.